app/lib/renderers/elements/shape.dart

```diff
@@ -9,6 +9,47 @@ class ShapeRenderer extends Renderer<ShapeElement> {
 
   ShapeRenderer(super.element, [super.layer]);
 
+  /// Creates a dashed path from the source path based on stroke style
+  Path _createDashedPath(Path source, StrokeStyle strokeStyle) {
+    if (strokeStyle == StrokeStyle.solid) return source;
+
+    final strokeWidth = element.property.strokeWidth;
+    final dashLength = strokeStyle == StrokeStyle.dashed
+        ? strokeWidth * 3 // Dashed: 3x stroke width
```

Comment:
I want to make the app as customizable as it can. I dont know if hard coded values make sense